### DISCUSSION TITLE
Automatic installer for yt-dlp-gui

### DISCUSSION THREAD

**Discussion by kazukikasama** [kannagi0303/yt-dlp-gui#62] [Show and tell]
Hello, i made an Automatic installer for yt-dlp-gui and Requirements FFMPEG, YT-DLP
[Auto installer Proyect](https://github.com/kazukikasama/youtube-dlp-gui-installer)

Quick download link, open in PowerShell
`iwr -useb https://raw.githubusercontent.com/kazukikasama/youtube-dlp-gui-installer/main/yt-dlp-gui-installer.ps1|iex`
@kazukikasama

**Comment by kannagi0303** [comment]
Thank you very much.
Glad you provided a convenient installation solution for this project.  :)

I tested it, and it's fairly simple to use.

If you'd like, can I include an entry "Third Party Installer" in my description and reference your project?

**Reply by kazukikasama** [reply]
Thank you, I really like the idea that you can include my installer :)

**Reply by kannagi0303** [reply]
I have updated the information to the Readme and Wiki, thank you again for the project.
